### DISCUSSION TITLE
Home assistant con disco dedicato a MotionEye

### DISCUSSION THREAD

**Discussion by opendore** [indomus/forum#524] [HUB: Home Assistant]
Ciao a tutti,

ho un'installazione di home assistant su un miniPC INtel con processore J1900.
Home Assistant è installato su un modulo mSata e funziona tutto correttamente.
Questo PC ha la possibilità di aggiungere un disco SATA, quello che vorrei fare è poter far vedere ad home assistant questo secondo disco da 1Tb per poterci salvare i filmati/snapshot catturati da MotionEye.
Come posso fare? Ho cercato da tempo ma non trovo una soluzione.

Grazie a tutti in anticipo.

**Comment by opendore** [comment]
...nessuno che sa aiutarmi?

Grazie

**Comment by emanueletocci** [comment]
Seguo... ho lo stesso problema

**Comment by Franco2255** [comment]
preso da indomus dove trovi tutta la guida

Storage Device / Root Directory | Si tratta della voci che permettono di decidere dove salvare i contenuti. Su “Storage Device” (selezionando “Custom Path“) è possibile scegliere un percorso specifico impostato poi nella voce “Root Directory“. Se si vuole utilizzare un disco locale collegato al computer è necessario collegarlo prima dell’avvio di motionEye e poi selezionarlo nella lista che appare nell’elenco “Storage Device“. Se si vuole utilizzare un percorso di rete (per esempio una condivisione Samba/CIFS di rete su NAS, o altro) è necessario impostare “Storage Device” su “Custom Path” e nel campo “Root Directory” impostare il percorso.

**Reply by opendore** [reply]
Ciao Franco2255, grazie per averci risposto.
Il mio problema è che ho un minipc con a bordo HomeAssistantOS e non so come fargli vedere un disco esterno per poi dedicarlo a MotionEye.
Hai un'idea?
Grazie in anticipo per il supporto.

**Reply by Franco2255** [reply]
Ciao io ho installato HomeAssistant su raspberry py4 con linux server ,
da quello che leggo HomeAssistantOS dovrebbe essere base linux e quindi se segui una guida su come montare il secondo hd su linux dovrebbe funzionare, non ne sono sicuro fai delle prove.
